src/Filters/LinearKalmanFilter.h

```diff
@@ -0,0 +1,57 @@
+#ifndef LINEAR_KALMAN_FILTER_H
+#define LINEAR_KALMAN_FILTER_H
+
+#include "Filter.h"
+#include "../Math/Matrix.h"
+
+namespace mmfs {
+
+class LinearKalmanFilter : public Filter {
+public:
+
+    int measurementSize;
+    int controlSize;
+    int stateSize;
+
+    // Constructors
+    LinearKalmanFilter(int measurementSize, int controlSize, int stateSize);
+    LinearKalmanFilter(Matrix X, Matrix P, Matrix K);
```

Comment:
passing kalman gain?

Reply:
yeah, I mean it's in there as an optional constructor, realistically they would just use `LinearKalmanFilter(Matrix X, Matrix P)` though

Reply:
yea I think we should get rid of it. passing in kalman gain is very odd to me and I don't think it will ever be used and might just cause confusion.